app/views/posts/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<section>
+  <div class="post-details">
+    <div class="post-title">
+      <h2><%= @post.title %> by <%= @post.author.name %></h2>
+      <div class="user-count">
+        <span>Comments: <%= @post.comments.count %></span>
+        <span>Likes: <%= @post.likes.count %></span>
+      </div>
+    </div>
+    <p><%= @post.text %></p>
+  </div>
+
+   <div class="comment">
+    <%= form_with(model: Comment.new, url: user_post_comments_path(@post.author, @post)) do |form| %>
+      <%= form.text_area :text, placeholder: 'Enter your comment', rows: 4 %>
+      <%= form.submit 'Add Comment' %>
+    <% end %>
+  </div>
+
```

Comment:
- [ ] Please make sure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments :pray:

Reply:
Hi @Owusu-Desmond  Will you please elaborate about this required Changes as this is not specified in the project requirement and I don't understood your required changes, will you show me some example about this. Thanks